Resources/Computing/ARCComputingElement.py

```diff
@@ -180,6 +181,9 @@ def submitJob( self, executableFile, proxy, numberOfJobs = 1 ):
     """ Method to submit job
     """
 
+    # Assume that the ARC queues are always of the format nordugrid-<batchSystem>-<queue>
+    # And none of our supported batch systems have a "-" in their name
+    self.arcQueue = self.queue.split("-",2)[2]
```

Comment:
And response from the ARC developers below ... so, this is fine I suppose, unless the siteDirector gets the queue wrong (can it?).

============
Short answer:

I don't foresee this is going to be changed in the lifetime of these
systems, so you might go ahead and hardcode such naming structure.